patches.json

```diff
@@ -241,6 +241,9 @@
     },
     "Fix PayPal issue with region": {
       "2.3.4": "MC-31387__fix_paypal_issue_with_region__2.3.4.patch"
+    },
+    "FPC is getting disabled during deployments": {
+      ">=2.3.2 <2.4.0": "MAGECLOUD-5069__fpc_is_getting_disabled_during_deployments__2.3.2.patch"
```

Comment:
This fix won't be delivered into 2.3.x?

Reply:
No, there are no plans to deliver it to 2.3 as far as I know.